packages/node/package.json

```diff
@@ -63,13 +63,16 @@
     "@opentelemetry/instrumentation-graphql": "0.39.0",
     "@opentelemetry/instrumentation-hapi": "0.36.0",
     "@opentelemetry/instrumentation-http": "0.48.0",
+    "@opentelemetry/instrumentation-ioredis":  "0.40.0",
     "@opentelemetry/instrumentation-koa": "0.39.0",
     "@opentelemetry/instrumentation-mongodb": "0.39.0",
     "@opentelemetry/instrumentation-mongoose": "0.37.0",
     "@opentelemetry/instrumentation-mysql": "0.37.0",
     "@opentelemetry/instrumentation-mysql2": "0.37.0",
     "@opentelemetry/instrumentation-nestjs-core": "0.36.0",
     "@opentelemetry/instrumentation-pg": "0.40.0",
+    "@opentelemetry/instrumentation-redis": "0.39.0",
+    "@opentelemetry/instrumentation-redis-4": "0.39.0",
```

Comment:
The packages are added, but not yet instrumented. Those are yet to be implemented in another PR.

Reply:
let's not add them yet, let's just add them when we introduce the respective integration!